omnigibson/configs/fetch_behavior.yaml

```diff
@@ -61,8 +59,11 @@ robots:
     controller_config:
       base:
         name: DifferentialDriveController
+      trunk:
+        name: JointController
```

Comment:
I think we shouldn't need this right? Can we remove it? From all of these files?

Reply:
this is more just to make it clear that the Fetch robot has a trunk that may be controlled